.github/workflows/ci.yml

```diff
@@ -183,5 +182,96 @@ jobs:
       - name: Publish Manifest File
         uses: actions/upload-artifact@v3
         with:
-          name: k8s_manifest
+          name: k8s_manifest_nonprod
+          path: ${{ env.K8S_RESOURCE_FILE }}
+
+  AppInfraProd:
+    if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main'
+    needs: DeployDev
```

Comment:
Dev won't deploy on main/master?

Reply:
Dev is built and deployed to on any push, but prod is only deployed to wiith main/master branch builds. This is aligned with the ADO criteria, right?

Reply:
Currently Stacks only builds dev on branches, not master/main.

Reply:
The current condition for Dev in the air-api.yml pipeline has no criteria for build/appinfraDev/deployDev. I am happy to implement those criteria if you feel strongly, and amend the criteria in the GH pipeline. Thoughts?